drape_frontend/apply_feature_functors.cpp

```diff
@@ -403,10 +403,10 @@ double BaseApplyFeature::PriorityToDepth(int priority, drule::rule_type_t ruleTy
   {
     if (depth < drule::kBasePriorityBgTop)
     {
-      ASSERT(ruleType == drule::area, (m_f.GetID()));
-      ASSERT_GREATER_OR_EQUAL(depth, drule::kBasePriorityBgBySize, (m_f.GetID()));
+      ASSERT(ruleType == drule::area, (m_f.DebugString()));
+      ASSERT_GREATER_OR_EQUAL(depth, drule::kBasePriorityBgBySize, (m_f.DebugString()));
```

Comment:
Do you really need to explicitly specify `.DebugString()`? Won't it work with just `m_f`?

Reply:
No, this magic works with `.DebugPrint()` only.
But `.DebugPrint()` requires its arg to be `const` whereas FeatureType is lazy loaded.
I'll add a comment to make it clear.